src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -52,6 +52,79 @@ using namespace orion;
 
 
 
+typedef struct CompoundSave
+{
+  std::string                name;
+  ContextAttribute*          aP;
+  orion::CompoundValueNode*  compoundP;
+  ContextAttribute*          aParentP;
+  std::string                state;
+} CompoundSave;
+
+static CompoundSave compoundSaveV[100];
```

Comment:
100 should be unhardwired to a #define (maybe in this same file, no need to include it in global.h in the case of a workaround)